pyomo/solvers/plugins/solvers/gurobi_direct.py

```diff
@@ -148,13 +148,31 @@ def _apply_solver(self):
                     raise
                 self._solver_model.setParam(key, float(option))
 
+        binary_vars = ComponentSet()
+        integer_vars = ComponentSet()
+        if 'relax_integrality' in self.options:
+            if self.options['relax_integrality']:
+                for v in self._solver_model.getVars():
+                    if v.vtype == self._gurobipy.GRB.BINARY:
+                        binary_vars.add(v)
+                        v.vtype = self._gurobipy.GRB.CONTINUOUS
+                    elif v.vtype == self._gurobipy.GRB.INTEGER:
+                        integer_vars.add(v)
+                        v.vtype = self._gurobipy.GRB.CONTINUOUS
+                self._solver_model.update()
```

Comment:
Question... are the bounds correctly set when changing the vtype to `GRB.CONTINUOUS`?  I only ask because I was once burned by this in Pyomo where simply changing the domain "forgot" the original bounds for Binary variables...